### PR TITLE
Add changelog to track changes, Closes #129

### LINKED ISSUE #129: Track changelog
Similar to the [RDF concepts spec](https://w3c.github.io/rdf-concepts/spec/#changes-12), we should keep track of the changes between 1.1 and 1.2.

### PR DESCRIPTION
**Pull request by rubensworks**
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/sparql-query/pull/146.html" title="Last updated on Apr 26, 2024, 11:04 AM UTC (6932845)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/sparql-query/146/4806e12...6932845.html" title="Last updated on Apr 26, 2024, 11:04 AM UTC (6932845)">Diff</a>

### CODE REVIEW

File: spec/index.html [hunk 1]
```diff
@@ -11799,6 +11799,58 @@ <h2>Internet Media Type, File Extension and Macintosh File Type</h2>
...
     </section>
     
+    <section id="changes-1-1" class="appendix informative">
+      <h2>Changes between SPARQL 1.1 Query Language and SPARQL 1.2 Query Language</h2>
+      <ul>
+        <li>
+            Fix errata:
+            <ul>
+              <li><a href="https://www.w3.org/2013/sparql-errata#editorial-query-1">editorial-query-1</a>: Missing right parenthesis in <a href="#defn_evalGraph">Evaluation of Graph definition</a></li>
+              <li><a href="https://www.w3.org/2013/sparql-errata#editorial-query-2">editorial-query-1</a>: Missing space in <a href="#defn_algJoin">Join definition</a></li>
```
Comment (Tpt) [inline spec/index.html:11809]: editorial-query-1 -> editorial-query-2
```suggestion
              <li><a href="https://www.w3.org/2013/sparql-errata#editorial-query-2">editorial-query-2</a>: Missing space in <a href="#defn_algJoin">Join definition</a></li>
```

File: spec/index.html [hunk 2]
```diff
@@ -11799,6 +11799,58 @@ <h2>Internet Media Type, File Extension and Macintosh File Type</h2>
...
+        <li>Improve display on mobile</li>
+        <li>Update grammar for quoted triples and triple functions in <a href="#sparqlGrammar" class="sectionRef"></a></li>
+        <li>Move RDFterm-equal and sameTerm to <a href="#func-rdfTerms" class="sectionRef"></a></li>
+        <li>Migrate XML Schema references to 1.1</li>
+        <li>Update references to XPath from 2.0 to 3.1</li>
+        <li>Add functions on quoted triples to <a href="#func-quoted-triples" class="sectionRef"></a></li>
+        <li>Add note on deduplication of triples produced by CONSTRUCT to <a href="#construct" class="sectionRef"></a></li>
+        <li>Remove historical notes on rdf:langString datatype from <a href="#func-datatype" class="sectionRef"></a></li>
+        <li>Remove inconsistencies between the definitions of the set functions</li>
+        <li>Introduce a function called multiplicity to replace card[Ω](μ) to <a href="#BasicGraphPattern" class="sectionRef"></a></li>
```
Comment (Tpt) [inline spec/index.html:11840]:
```suggestion
        <li>Introduce a function called multiplicity to replace card[Ω](μ) in <a href="#BasicGraphPattern" class="sectionRef"></a></li>
```

**Review by Tpt — APPROVED** [review]

**Comment by hartig** [comment]
Thanks for your effort @rubensworks !!

I wonder, however, whether the list can be organized a bit better. Currently, it looks quite random. Maybe it is possible to group the changes (e.g., 'minor editorial changes', 'fixes', ?) and perhaps also to order them in terms of the parts of the spec that they have touched.

**Review by domel — APPROVED** [review]

**Comment by rubensworks** [comment]
> I wonder, however, whether the list can be organized a bit better. Currently, it looks quite random. Maybe it is possible to group the changes (e.g., 'minor editorial changes', 'fixes', ?) and perhaps also to order them in terms of the parts of the spec that they have touched.

Sure, I'll look into some more grouping. Already did it for the errata, but wasn't sure about the others.

**Review by hartig — APPROVED** [review]
Thanks for adding the categories!

**Review by afs — APPROVED** [review]

**Pull request merged**